elasticsearch.md

```diff
@@ -1501,63 +1501,178 @@ ReadonlyREST can write events very similarly to Logstash into to a series of ind
   }
 ```
 
-Here is a configuration example, you can see the `audit.collector: true` setting, which normally defaults to false. Note how the successful requests matched by the first rule \(Kibana\) will not be written to the audit log, because the verbosity is set to error. Audit log in facts, obey the verbosity setting the same way regular text logs do.
+#### Configuration
```

Comment:
I want to ask you about one more refactor. The configuration part of the auditing feature is huge. Let's move it to the `details` folder to a separate file and link it here.

Please, don't forget by modifying SUMMARY.md (for a gitbook's sake)